src/Advertising.js

```diff
@@ -212,6 +212,9 @@ export default class Advertising {
   }
 
   defineSlots() {
+    if (!this.config.slots) {
```

Comment:
I've added this, it was a bug: `slots` is supposed to be optional, but we didn't null-check it